llama_index/llms/localai.py

```diff
@@ -1,75 +0,0 @@
-from typing import Any, Dict, Optional
-
-from llama_index.bridge.pydantic import Field
-from llama_index.constants import DEFAULT_CONTEXT_WINDOW
-from llama_index.llms.base import LLMMetadata
-from llama_index.llms.openai import OpenAI
-from llama_index.llms.openai_utils import is_function_calling_model
-
-DEFAULT_API_KEY = "fake"
-DEFAULT_API_HOST = "localhost"
-DEFAULT_API_PORT = 8080
-DEFAULT_API_BASE = f"{DEFAULT_API_HOST}{DEFAULT_API_PORT}"
-
-
-class LocalAI(OpenAI):
```

Comment:
This is now deprecated in favor of the more generalized `OpenAILike`, thanks @benbot for enabling this